### ISSUE TITLE
Small words separated by slash are capitalized

### ISSUE THREAD

**Issue by AlejandroFrias** [ppannuto/python-titlecase#21]
Because of the recursive call to `titlecase` in the case where the word is separated by a single slash, small words get capitalized as they are the first word in the string (the only word really).

So:
```
>>> titlecase("a title and/or string")
"A Title And/Or String"
```
instead of 
```
>>> titlecase("a title and/or string")
"A Title and/or String"
```

Could fix this with an optional flag to ignore the rule about starting with a small word that is set in the recursive call only. Or explicitly check for small words again on the split word. I prefer the flag method because this will reapply the callback and any other interesting logic to each word.

**Comment by ppannuto** [comment]
I agree that the flag approach makes more sense - happy to take a PR that implements it!

**Comment by ppannuto** [comment]
Fixed by #23
